src/requests/getForecast.js

```diff
@@ -1,40 +0,0 @@
-/* eslint-disable-next-line no-console */
-import axios from "axios";
-
-const getForecast = (
-  setSelectedDate,
-  setForecasts,
-  setLocation,
-  searchText,
-  setErrorMessage
-) => {
-  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
-
-  if (searchText) {
-    endpoint += `?city=${searchText}`;
-  }
-
-  axios
-    .get(endpoint)
-    .then((response) => {
-      setSelectedDate(response.data.forecasts[0].date);
-      setForecasts(response.data.forecasts);
-      setLocation(response.data.location);
-      setErrorMessage("");
-    })
-    .catch((error) => {
-      const { status } = error.response || {};
-      if (status === 404) {
-        setErrorMessage(`${searchText}? Never heard of it!`);
```

Comment:
funny, but I'd add the formal description of what the error along with the more casual message. Something like:

Never heard of it!

error: location not found